UnitTests/GitCommandsTests/Git/GitBranchNameNormaliserTest.cs

```diff
@@ -18,16 +18,15 @@ public void Setup()
         }
 
 
-        [TestCase("[BUG-1234] some long description located [//test/*/dir?/.lock]{3,} and <foo:bar\\can>...", "_", "_BUG-1234]_some_long_description_located__/test/_/dir_/_lock]{3,}_and__foo_bar_can__")]
-        [TestCase("[BUG-1234] some long description located [//test/*/dir?/.lock]{3,} and <foo:bar\\can>...", "-", "-BUG-1234]-some-long-description-located--/test/-/dir-/-lock]{3,}-and--foo-bar-can--")]
-        [TestCase("[BUG-1234] some long description located [//test/*/dir?/.lock]{3,} and <foo:bar\\can>...", "", "BUG-1234]somelongdescriptionlocated/test/dir/lock]{3,}andfoobarcan")]
+        [TestCase("[BUG-1234] some long description located [//test/*/dir?/.lock]{3,} and foo:bar\\can...", "_", "_BUG-1234]_some_long_description_located__/test/_/dir_/_lock]{3,}_and_foo_bar_can_")]
+        [TestCase("[BUG-1234] some long description located [//test/*/dir?/.lock]{3,} and foo:bar\\can...", "-", "-BUG-1234]-some-long-description-located--/test/-/dir-/-lock]{3,}-and-foo-bar-can-")]
+        [TestCase("[BUG-1234] some long description located [//test/*/dir?/.lock]{3,} and foo:bar\\can...", "", "BUG-1234]somelongdescriptionlocated/test/dir/lock]{3,}andfoobarcan")]
```

Comment:
Windows and *nix-based systems have different opinions as to what an invalid char is (https://github.com/mono/mono/blob/master/mcs/class/corlib/System.IO/Path.cs#L570)
It feels like too much hassle to account for each platform...